images/gh_runner/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Cache Holodeck image
-FROM ghcr.io/nvidia/holodeck:v0.2.1 AS holodeck
+FROM ghcr.io/nvidia/holodeck:v0.2.3 AS holodeck
```

Comment:
@ArangoGutierrez it seems strange that this is bumped separately from the release? Could you just clarify how the different versions are related?

Reply:
Ok. Sorry, this is my bad. Makes sense.